ERCS/erc-7556.md

```diff
@@ -0,0 +1,157 @@
+      schema:
+        type: string
+      description: The chain_id of a given netowrk.
+    - in: query
+      name: plugin_address
+      schema:
+        type: string
+      description: The address of the ERC-6900 plugin.
+    - in: query
+      name: manifestHash
+      schema:
+        type: string
+      description: The hash of the plugin manifest.
+    - in: query
+      name: pluginInitData
+      schema:
+        type: string
+      description: The data to be decoded and used by the plugin to setup initial plugin data for the modular account. At a minimum this must include the embedded wallet's public address.
+    - in: query
+      name: dependencies
+      schema:
+        type: string
+      description: The dependencies of the plugin, as described in the manifest.
+    - in: query
+      name: injectedHooks
+      schema:
+        type: string
+      description: Optional hooks to be injected over permitted calls this plugin may make.
+```
+
+##### Response
+```=
+ parameters:
+    - in: query
+      name: result
+      schema:
+        type: boolean
+      description: Returns a true if the plugin was successfully registered.
+    - in: query
+      name: plugin_address
+      schema:
+        type: boolean
+      description: The address of the ERC-6900 plugin that was registered.
+    - in: query
+      name: smart_account_address
+      schema:
+        type: boolean
+      description: The address of the smart account where the plugin was registered.
+```
+
+#### Syntax
+##### URI Request Syntax
+```=
+https://<PROVIDER_URI>/
+    redirect_uri=<YOUR_REDIRECT_URI>
+    &smart_account_address=<USER_SMART_ACCOUNT>
+    &plugin_address=<PLUGIN_ADDRESS>
+    &manifest_hash=<MANIFEST_HASH>
+    &plugin_init_data<PLUGIN_INIT_DATA>
+    &dependencies=<DEPENDENCIES>
+    &injected_hooks=<INJECTED_HOOKS>
+```
+
+##### URI Response Syntax
+```=
+https://<YOUR_REDIRECT_URI>?
+    result=<RESULT_BOOLEAN>
+    &plugin_address=<PLUGIN_ADDRESS>
+    &smart_account_address=<SMART_ACCOUNT_ADDRESS>
+```
+
+## Rationale
+#### Application
+##### Initial Request
+An application must provide all relevant information that is required to successfully call `installPlugin` per ERC-6900, the `smart_account_address` and the `chain_id` are added to the request to ensure the plugin is install on the correct smart account. Using ERC-XXXX a developer would have already discovered the `smart_account_address` of a given user.
+
+##### Response from provider
+An application simply needs to check the result of the request, and act on it accordingly.
+
+#### Provider
+Upon parsing all the relevant data, it is the providers job to correctly attempt to install the plugin, if it fails do so, the interface should alert the user why it was not able to do so. If the plugin is already installed, the provider should simply return successfully.
+
+## Backwards Compatibility
+
+No backward compatibility issues found.
+
+## Reference Implementation
+// TODO
```

Comment:
```suggestion
<!--
## Reference Implementation
// TODO
-->
```

The reference implementation section isn't mandatory, so you can just comment it out.